driver/main.cpp

```diff
@@ -72,6 +72,7 @@ int main(int argc, char* argv[])
         std::cout << "ParseCmdLineArgs() FAILED, rc = " << rc << std::endl;
         return rc;
     }
+
```

Comment:
Please avoid to add irrelevant modification to the pull request, especially when we don't have the ownership of this project.

Reply:
I have removed modification at [6f6fe652](https://github.com/moreh-dev/MIOpen/pull/6/commits/6f6fe652a6f21e1e315ea829f9d3927e5a4085e2)